src/user_pref_controller.py

```diff
@@ -22,20 +23,33 @@ class UserPrefController(UserPrefInterface):
     #   True or Value means success
     #   False or exception means did not succeed
     # Unless otherwise Specified
-    def __init__(self, database_name:str="user_pref.db", database_location:pathlib.Path=pathlib.Path.home()) -> None:
-        """ Creating dependent classes and setting up the file system for the user """
+    def __init__(self,
+                 database_name: str = "user_pref.db",
+                 database_location: pathlib.Path = pathlib.Path.home()
+                 ) -> None:
+        """
+        Creating dependent classes and setting up the file system for the user
+        """
         logger.info("START: UserPreferences Database")
-        logger.debug("database_location: %s, database_name: %s", database_location, database_name)
-        self.db_location:pathlib.Path = database_location / ".onko" # database directory ( "." makes file hidden in linux and macOS)
+        logger.debug(
+            "database_location: %s, database_name: %s",
+            database_location, database_name
+        )
+        # database directory ( "." makes file hidden in linux and macOS)
+        self.db_location: pathlib.Path = database_location / ".onko"
```

Comment:
one would typically use database_location.join(".onko"), but if this works, it's good enough.